packages/web/src/components/collection/desktop/CollectionHeader.tsx

```diff
@@ -1,42 +1,87 @@
-import { ChangeEvent, useCallback, useState } from 'react'
+import { ChangeEvent, MouseEventHandler, useCallback, useState } from 'react'
 
 import { useGetCurrentUserId } from '@audius/common/api'
+import {
+  AccessConditions,
+  AccessPermissions,
+  CoverArtSizes,
+  ID,
+  Variant,
+  isContentUSDCPurchaseGated
+} from '@audius/common/models'
 import { FeatureFlags } from '@audius/common/services'
 import {
+  CollectionsPageType,
   PurchaseableContentType,
   useEditPlaylistModal
 } from '@audius/common/store'
 import { formatSecondsAsText, formatDate } from '@audius/common/utils'
 import {
   Text,
-  IconFilter,
   IconVisibilityHidden,
   IconPencil,
-  Flex
+  Flex,
+  TextInput,
+  TextInputSize,
+  IconSearch,
+  IconCart,
+  useTheme,
+  IconComponent
 } from '@audius/harmony'
 import cn from 'classnames'
 
 import { ClientOnly } from 'components/client-only/ClientOnly'
-import { Input } from 'components/input'
 import { UserLink } from 'components/link'
-import RepostFavoritesStats from 'components/repost-favorites-stats/RepostFavoritesStats'
 import Skeleton from 'components/skeleton/Skeleton'
 import { GatedContentSection } from 'components/track/GatedContentSection'
-import InfoLabel from 'components/track/InfoLabel'
 import { UserGeneratedText } from 'components/user-generated-text'
 import { useFlag } from 'hooks/useRemoteConfig'
 import { useSsrContext } from 'ssr/SsrContext'
 
 import { Artwork } from './Artwork'
 import { CollectionActionButtons } from './CollectionActionButtons'
 import styles from './CollectionHeader.module.css'
+import { RepostFavoritesStats } from './RepostsFavoritesStats'
 
 const messages = {
-  filter: 'Filter Tracks'
+  filter: 'Search in playlist...'
```

Comment:
is it always a playlist?

Reply:
Oop no good catch; this was just to show design and I forgot to fix it